frontend/src/app/create-employee/create-employee.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { EmployeesService } from '../api/employees.service';
+import { NewEmployee } from '../model/newEmployee';
+
+@Component({
+  selector: 'app-create-employee',
+  templateUrl: './create-employee.component.html',
+  styleUrls: ['./create-employee.component.css']
+})
+export class CreateEmployeeComponent implements OnInit {
+
+  constructor(private employeeServvice: EmployeesService) { }
+
+  ngOnInit(): void {
+  }
+
+  createEmployee(name: string, attr: string): void {
+    var newemployee: NewEmployee = {
```

Comment:
Please comply to camelCase naming convention.
And in typescript, we do not use the keyword "var"
If something is a constant, use "const", if something is a variable, use "let"